### DISCUSSION TITLE
Problemas inicio de sesión con el correo

### DISCUSSION THREAD

**Discussion by CometArao** [Launch-X-Latam/MisionFrontEnd#231] [General]
Buenas, con quién me puedo poner en contacto para solucionar un problema de inicio de sesión de mi correo? Ya que me sacó la sesión del Teams y no me está dejando iniciar con mi contraseña

**Comment by doguedogue** [comment]
contacto@innovaccion.mx

**Reply by CometArao** [reply]
Desde hace tiempo que no me responden. ¿Hay algún encargado directo con el que poder charlar?

**Reply by Cerecero** [reply]
@CometArao Intenta con un mensaje por Instagram, ese tambien es un canal de comunicacion.
